packages/flutter_tools/test/general.shard/windows/visual_studio_test.dart

```diff
@@ -717,7 +734,7 @@ void main() {
       expect(visualStudio.fullVersion, equals('16.2.29306.81'));
     });
 
-    testWithoutContext('cmakePath returns null when VS is present but when vswhere returns invalid JSON', () {
+    testWithoutContext('Warns when VS is present but vswhere returns invalid JSON', () {
```

Comment:
Nit: Warns and returns no installation when [...]

Reply:
Fixed!